src/ui/public/modals/confirm_modal.html

```diff
@@ -1,32 +1,10 @@
-<div class="kuiModal" style="width: 450px" data-test-subj="confirmModal">
-  <div class="kuiModalHeader" ng-if="title">
-    <div class="kuiModalHeader__title" data-test-subj="confirmModalTitleText">
-      {{title}}
-    </div>
-  </div>
-  <div class="kuiModalBody">
-    <div
-      class="kuiModalBodyText"
-      data-test-subj="confirmModalBodyText"
-    >
-      {{message}}
-    </div>
-  </div>
-
-  <div class="kuiModalFooter">
-    <button
-      class="kuiButton kuiButton--hollow"
-      data-test-subj="confirmModalCancelButton"
-      ng-click="onCancel()"
-    >
-      {{cancelButtonText}}
-    </button>
-    <button
-      class="kuiButton kuiButton--primary"
-      data-test-subj="confirmModalConfirmButton"
-      ng-click="onConfirm()"
-    >
-      {{confirmButtonText}}
-    </button>
-  </div>
-</div>
+<confirm-modal
```

Comment:
> I say we move it out of this specific PR since it's not something new that this PR introduced

Agreed. I'm very much against auto-loading modules on startup, we used to do that, and it got very messy; explicit imports are much easier to understand. I don't know why the auto-loading stuff has come back and why there hasn't been a strong push against it. But that's not a conversation for this PR. I was just trying to understand our plans around the Angular+React stuff.